frontend/src/component/Account/UserInfo.styl

```diff
@@ -15,7 +15,8 @@
       width 100px
       height 100px
   &__name
-    font-size 22px
+    font-size 24px
```

Comment:
Can you use one of the font size variables that we have ?
See `frontend_lib/src/css/Variable.styl`

Reply:
Thanks @gbonaspetti. Yes I can. If I understand correctly, font-size 22 px does not exist either (it was the previous font size) ?

Reply:
Yes, an old hardcoded value, I think the best for your case is fontSizeTitle